Sources/TuistKit/GraphMappers/CacheWorkspaceMapper.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import TuistCore
+import TuistGraph
+
+final class CacheWorkspaceMapper: WorkspaceMapping { // swiftlint:disable:this type_name
```

Comment:
The name of the mapper doesn't say much about what it does. What about renaming it to `GenerateCacheableSchemesWorkspaceMapper`. I know it's a bit longer, but you know what it does without having to look at the implementation.

Reply:
Also, I'd add a comment summarizing what the mapper is for.